### ISSUE TITLE
Ontopoly on Google App Engine

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ontopia/ontopia#124]
```
I have made a quick test of running Ontopoly as a part of Ontopia-
Stormberg-demowebapp on GAE.
Here's the instructions for getting started: http://
stronglytypedblog.blogspot.com/2009/04/wicket-on-google-app-engine.html

The problems:
OntopolyApplication and ontopoly.LockManager are not serializable.

And
java.lang.NoClassDefFoundError: javax.swing.tree.DefaultTreeModel is a 
restricted class. Please see the Google App Engine developer's guide for 
more details.
In other words, swingstuff is blacklisted!
```

Original issue reported on code.google.com by `stig....@gmail.com` on 24 Aug 2009 at 6:47

Attachments:
- [stacktrace.txt](https://storage.googleapis.com/google-code-attachments/ontopia/issue-124/comment-0/stacktrace.txt)
